apps/desktop-wallet/src/App.tsx

```diff
@@ -88,6 +86,7 @@ const App = () => {
   const showDevIndication = useDevModeShortcut()
   const posthog = usePostHog()
   const { sendAnalytics } = useAnalytics()
+  useAlphPrice()
```

Comment:
Prefetching ALPH price to re-create the previous behavior

Reply:
Mhh. I'm not sure I like it. Does it really bring value over fetching the price in the component that is first mounted and which requires the price?